doc/source/user_guide/plots_arguments_guide.md

```diff
@@ -151,23 +165,44 @@ Plot with an increased text scaling factor based on `textsize`:
 az.plot_posterior(data, var_names=["mu"], textsize=3);
 ```
 
+(common_color)=
+## `color`
+
+Color used for the limits of the HDI and fill. Should be a valid matplotlib color.
```

Comment:
Do you think it would be better to just leave this out of the plotting arguments guide since it varies so much across plotting functions? And relegate explanations to the docs page for each plotting function?

Reply:
We can take this one from the common list yeah, like with kind